ft/onto/base_ontology.py

```diff
@@ -555,6 +594,73 @@ class AudioUtterance(AudioAnnotation):
 
     speaker: Optional[str]
 
-    def __init__(self, pack: PackType, begin: int, end: int):
+    def __init__(self, pack: DataPack, begin: int, end: int):
         super().__init__(pack, begin, end)
         self.speaker: Optional[str] = None
+
+
+@dataclass
+class AudioPayload(Payload):
+    """
+    A payload that caches audio data
+    Attributes:
+        sample_rate (Optional[int]):
+    """
+
+    sample_rate: Optional[int]
+
+    def __init__(
+        self,
+        pack: DataPack,
+        modality: IntEnum,
```

Comment:
We need to somehow remove the "modality" attribute here, otherwise, this interface is too redundant for the user.